relay-server/src/actors/envelopes.rs

```diff
@@ -201,15 +198,29 @@ impl FromMessage<SendMetrics> for EnvelopeManager {
 #[derive(Debug)]
 pub struct EnvelopeManagerService {
     config: Arc<Config>,
+    aggregator: Addr<Aggregator>,
+    enveloper_processor: Addr<EnvelopeProcessor>,
+    test_store: Addr<TestStore>,
+    upstream_relay: Addr<UpstreamRelay>,
```

Comment:
nit: Should we make this an "address book" again?

Reply:
This for me still fine and I would keep it separated for now and I would like to keep this still as it is.